### PR TITLE
Issue #3180882: Additional breadcrumb link on node/add pages

### PR DESCRIPTION
**Pull request by ol0lll**
…erview

Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * Class to define the breadcrumb builder.
+ */
+class ThunderNodeEditBreadcrumbBuilder implements BreadcrumbBuilderInterface {
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(RouteMatchInterface $route_match) {
+    return in_array($route_match->getRouteName(), [
```
Comment (dbosen) [inline src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php:21]: This does not work for me on node edit, this applies method is not even called on the edit form

**Review by dbosen — CHANGES_REQUESTED** [review]

**Review by dbosen — CHANGES_REQUESTED** [review]
This also needs tests

File: src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php (hunk 1 above)
Comment (dbosen) [reply src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php:21]: The reason is, that  ThunderArticleBreadcrumbBuilder.php wins in that case.
We should consolidate both builders

Comment (ol0lll) [reply src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php:21]: I disagree, ThunderArticleBreadcrumbBuilder should be more specific

File: composer.json [hunk 2]
```diff
@@ -38,6 +38,9 @@
     "extra": {
         "installer-name": "thunder",
         "patches": {
+            "drupal/core" : {
+                "PHPUnit 9.5 Call to undefined method ::getAnnotations()": "https://www.drupal.org/files/issues/2020-12-04/3186443-1.patch"
```
Comment (chrfritsch) [inline composer.json]: We have fixed to PHPUnit 9.4 for now. This can be removed

**Review by dbosen — APPROVED** [review]
looks good!

**Pull request merged**